kolibri/plugins/coach/assets/src/views/reports/ReportsQuizQuestionPage.vue

```diff
@@ -5,7 +5,7 @@
     :immersivePagePrimary="true"
     immersivePageIcon="back"
     :immersivePageRoute="toolbarRoute"
-    :appBarTitle="exam.title"
+    :appBarTitle="exam && exam.title"
```

Comment:
minor passing thought:

would be cool if we could use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

This might be more explicit than the boolean expression?

```suggestion
    :appBarTitle="exam ? exam.title : null"
```